ethereumj-core/src/main/java/org/ethereum/net/server/EthereumChannelInitializer.java

```diff
@@ -115,6 +92,43 @@ public void initChannel(NioSocketChannel ch) throws Exception {
             logger.error("Unexpected error: ", e);
         }
     }
+    
+    private boolean shouldRefuseIncoming(NioSocketChannel ch) {
```

Comment:
Maybe something like `isEligibleForConnection`?

Reply:
i like it!

Reply:
almost )